### PR TITLE
Add space to URL

### PR DESCRIPTION
**Pull request by benrudolph**
@czue seems like a hack solution, but should fix for people who have user ids that have a space in them. Going to try and make it so that can't happen again. http://manage.dimagi.com/default.asp?179101. Was able to repro locally

cc: @biyeun

### CODE REVIEW

File: corehq/apps/users/urls.py [hunk 1]
```diff
@@ -49,11 +49,11 @@
 patterns("corehq.apps.users.views.mobile.users",
     url(r'^commcare/$', ListCommCareUsersView.as_view(), name=ListCommCareUsersView.urlname),
     url(r'^commcare/fields/$', UserFieldsView.as_view(), name=UserFieldsView.urlname),
-    url(r'^commcare/account/(?P<couch_user_id>[\w-]+)/$', EditCommCareUserView.as_view(), name=EditCommCareUserView.urlname),
-    url(r'^commcare/account/(?P<couch_user_id>[\w-]+)/user_data/$', 'update_user_data', name='update_user_data'),
-    url(r'^commcare/account/(?P<couch_user_id>[\w-]+)/groups/$', 'update_user_groups', name='update_user_groups'),
+    url(r'^commcare/account/(?P<couch_user_id>[ \w-]+)/$', EditCommCareUserView.as_view(), name=EditCommCareUserView.urlname),
```
Comment (dimagimon) [inline corehq/apps/users/urls.py]: E501 line too long (126 > 115 characters)

File: corehq/apps/users/urls.py [hunk 2]
```diff
@@ -49,14 +49,14 @@
...
+    url(r'^commcare/account/(?P<couch_user_id>[ \w-]+)/user_data/$', 'update_user_data', name='update_user_data'),
+    url(r'^commcare/account/(?P<couch_user_id>[ \w-]+)/groups/$', 'update_user_groups', name='update_user_groups'),
     url(r'^commcare/list/$', AsyncListCommCareUsersView.as_view(), name=AsyncListCommCareUsersView.urlname),
-    url(r'^commcare/archive/(?P<user_id>[\w-]+)/$', 'archive_commcare_user', name='archive_commcare_user'),
-    url(r'^commcare/unarchive/(?P<user_id>[\w-]+)/$', 'archive_commcare_user', name='unarchive_commcare_user', kwargs={'is_active': True}),
-    url(r'^commcare/delete/(?P<user_id>[\w-]+)/$', 'delete_commcare_user', name='delete_commcare_user'),
-    url(r'^commcare/restore/(?P<user_id>[\w-]+)/$', 'restore_commcare_user', name='restore_commcare_user'),
+    url(r'^commcare/archive/(?P<user_id>[ \w-]+)/$', 'archive_commcare_user', name='archive_commcare_user'),
+    url(r'^commcare/unarchive/(?P<user_id>[ \w-]+)/$', 'archive_commcare_user', name='unarchive_commcare_user', kwargs={'is_active': True}),
```
Comment (dimagimon) [inline corehq/apps/users/urls.py]: E501 line too long (140 > 115 characters)

**Comment by czue** [comment]
wow i had no idea this worked:

image

what mechanism do we expose to let people override their couch IDs? bulk upload?

**Comment by czue** [comment]
nevermind, just saw https://github.com/dimagi/commcare-hq/pull/7884

**Pull request merged**